src/main/java/TaskList.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    private static final ArrayList<Task> TaskList = new ArrayList<>(10);
```

Comment:
ArrayList is a type of dynamic array and there is no need to limit your `TaskList` to a size of 10 only!